src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -190,7 +190,7 @@ object TezosTypes {
       gas_limit: PositiveBigNumber,
       storage_limit: PositiveBigNumber,
       public_key: PublicKey,
-      source: ContractId,
+      source: PublicKeyHash,
```

Comment:
Please remind me again why we have two distinct types called `ContractId` and `PublicKeyHash`?

Reply:
Semantics reason only. There's no real difference, only that at the time I wanted to differentiate the usage.